app/javascript/elm/src/Main.elm

```diff
@@ -871,19 +871,11 @@ setScrollPosition value =
 viewDocument : Model -> Browser.Document Msg
 viewDocument model =
     { title = "AirCasting"
-    , body = [ view model ]
+    , body = [ snippetGoogleTagManager 
+    ,  view model ]
```

Comment:
this is what likely causes the build failure, try running `yarn run elm-format --validate app/javascript/elm/` (and I think without the `--validate` to simply format the files properly)